doc/source/api/visualization/plotter_windows_manager.rst

```diff
@@ -7,43 +7,47 @@ The ``PlotterWindowsManager`` class provides for managing and directly interacti
 with Matplotlib windows. By registering these methods with ``EventsManager``, you can
 update plots during run time.
 
-The following example updates ``window-1`` and ``window-2`` during solution initialization 
+The following example updates ``p_xy`` and ``p_res`` during solution initialization
 and whenever data is read. During the calculation, it also updates both windows at the end
 of every time step.
 
 .. code-block:: python
 
-    from ansys.fluent.visualization import Plots
-    from ansys.fluent.visualization.plotter import plotter_windows_manager
+    from ansys.fluent.visualization import XYPlot, Monitor, GraphicsWindow
     
     plots_session = Plots(session)
     
     #Create xy plot.
-    plot1 = plots_session.XYPlots["plot-1"]
+    plot1 = XYPlot(solver=session)
     plot1.surfaces = ['symmetry']
     plot1.y_axis_function = "temperature"
     
     
-    # Plot XY plot on window-1.
-    plot1.plot("window-1")
+    # Plot XY plot.
+    p_xy = GraphicsWindow()
+    p_xy.add_graphics(plot1)
+    p_xy.show()
     
     #Create monitor plot.
-    monitor1 = plots_session.Monitors["monitor-1"]
+    monitor1 = Monitor(solver=session)
     monitor1.monitor_set_name = "residual"
     
     
-    #Plot monitor on window-2.
-    monitor1.plot("window-2")   
+    #Plot monitor.
+    p_res = GraphicsWindow()
+    p_res.add_graphics(monitor1)
+    p_res.show()
     
-    # Create callback that refreshes window-1 and window-2.    
-    def auto_refresh_plot(session_id, event_info):    
-        plotter_windows_manager.refresh_windows(session_id, ["window-1", "window-2"])
+    # Create callback that refreshes 'p_xy' and 'p_res'.
+    def auto_refresh_plot(session_id, event_info):
+        # 'p_xy' and 'p_res' uses 'plotter_windows_manager' in its implementation
+        p_xy.refresh_windows(session.id)
```

Comment:
`p_xy` is a window so `xy_window.refresh()` seems more appropriate -- dropping the `_windows` suffix.